app/views/content_items/guide_single.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :extra_head_content do %>
+  <%= machine_readable_metadata(
+    schema: :article,
+    canonical_url: @content_item.canonical_url,
+    body: @content_item.current_part_body
+  ) %>
+<% end %>
+
+<% if MachineReadable::YamlFaqPageSchemaPresenter.configured?(@content_item) %>
+<script type="application/ld+json">
+  <%= raw MachineReadable::YamlFaqPageSchemaPresenter.new(@content_item).structured_data.to_json %>
+</script>
+<% end %>
+
+<% content_for :simple_header, true %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', { title: @content_item.content_title } %>
+
+    <% if @content_item.show_guide_navigation? %>
+      <aside class="part-navigation-container" role="complementary">
+        <%= render "govuk_publishing_components/components/contents_list", aria_label: 'Pages in this guide', contents: @content_item.part_link_elements, underline_links: true %>
+      </aside>
+    <% end %>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <% if @content_item.has_parts? %>
+
+      <% @content_item.parts.each_with_index do |part, index| %>
+      <section class="govuk-!-padding-bottom-9" aria-labelledby="<%= part["slug"] %>">
```

Comment:
The padding class `govuk-!-padding-bottom-9` is not ideal, we could create a class for this section in future. We already used them widely across the app, it's not something to be changed in this PR.